lib/app_routes/getAllStudies.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 exports.register = function(server, options, next) {
-    var Study = require('../models/Study')(options.bookshelf);
+    var Study = server.plugins.bookshelf.model('Study');
```

Comment:
@ruwang @rsskmr,

just thinking out loud. `lib/app_routes/study.js` could be a simpler representation vs `lib/app_routes/getAllStudies.js`, where inside theres a `/study` and `/study/{id}` route handler exported.  this is how it works in the v1-v3 API system.  Route definition file `coins_core/api/index.php` map associated handlers `coins_core/api/handers/AnyHandlerName.php`.  this has worked really well.  we may have already discussed this.  i'd recommend at least looking into and considering the strategy